src/NamingStrategy/PathNamingStrategy.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Http\Client\Plugin\Vcr\NamingStrategy;
+
+use Psr\Http\Message\RequestInterface;
+use Symfony\Component\OptionsResolver\Options;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Will use the request path as filename.
+ *
+ * @author Gary PEGEOT <garypegeot@gmail.com>
+ */
+class PathNamingStrategy implements NamingStrategyInterface
+{
+    /**
+     * @var array
+     */
+    private $options;
+
+    public function __construct(array $options = [])
+    {
+        $resolver = new OptionsResolver();
+        $this->configureOptions($resolver);
+        $this->options = $resolver->resolve($options);
+    }
+
+    public function name(RequestInterface $request): string
+    {
+        $parts = [$this->options['name_prefix']];
```

Comment:
what if the prefix is not set? then the files willl start with `_`. what is the use case for the name prefix? i think i would not have this option and people should set the recorder to different directories.

Reply:
I'll remove it then